core/src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java

```diff
@@ -495,8 +496,8 @@ public void addSplits(String tableName, SortedSet<Text> partitionKeys)
     CountDownLatch latch = new CountDownLatch(splits.size());
     AtomicReference<Exception> exception = new AtomicReference<>(null);
 
-    ExecutorService executor =
-        context.threadPools().getPoolBuilder("addSplits").numCoreThreads(16).build();
+    ExecutorService executor = context.threadPools()
+        .getPoolBuilder(METRICS_POOL_PREFIX + "table.ops.add.splits").numCoreThreads(16).build();
```

Comment:
I don't think this thread pool will emit metrics because metrics are not being turned on in the builder.  So this change will just impact the name of the thread as seen in jstack, was that the intention?

Reply:
The goal was to try to be consistent across all names.   It would make is easier to recognize that it was a pool It seemed adopting a convention would help anywhere the name could show up, logs, jstacks,...

Reply:
Is it the case that all pool names will have `METRICS_POOL_PREFIX` in the leading portion of the name? If that's intended, maybe we should add that to the pool builder so it's done in one place?